web-gateway/app/controllers/ItemController.java

```diff
@@ -33,23 +35,30 @@
     private final BiddingService bidService;
 
     private final Boolean showInlineInstruction;
+    private HttpExecutionContext ec;
```

Comment:
Did you ask anyone on the Play team about the need to specify an explicit `HttpExecutionContext`? It seems like it has created a very ugly API 😦